### PR TITLE
[Debian 10] build_packages for debian 10

### PR DESCRIPTION
**Pull request by xlqian**
build_packages for debian 10

### CODE REVIEW

File: .github/workflows/build_navitia_packages_for_release_debian_10.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+on:
+  pull_request:
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    container:
+      image: navitia/debian10_dev
```
Comment (nonifier) [inline .github/workflows/build_navitia_packages_for_release_debian_10.yml]: Why not using a matrix build on the previous script ? So we can do `[navitia/debian8_dev, navitia/debian10_dev]`
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

File: .github/workflows/build_navitia_packages_for_dev.yml [hunk 2]
```diff
@@ -1,17 +1,20 @@
 name: Build Navitia Packages For Dev
 
 on:
-  push:
-    branches:
-      - dev
+  pull_request:
```
Comment (xlqian) [inline .github/workflows/build_navitia_packages_for_dev.yml]: just for test

**Comment by nonifier** [comment]
Do you plan on keeping the original file ? 
https://github.com/CanalTP/navitia/blob/dev/.github/workflows/build_navitia_packages_for_dev.yml

**Review by benoit-bst — APPROVED** [review]

**Comment by xlqian** [comment]
@nonifier we should remove the old one, once all CI jobs are migrated to the new one

**Review by nonifier — APPROVED** [review]

**Pull request merged**